ragna/assistants/_ai21labs.py

```diff
@@ -21,8 +21,8 @@ def _make_system_content(self, sources: list[Source]) -> str:
         )
         return instruction + "\n\n".join(source.content for source in sources)
 
-    async def _call_api(
```

Comment:
The `def _call_api` abstraction for `def answer` was just a remnant of an old implementation that I forgot to clean up earlier:

https://github.com/Quansight/ragna/blob/84cf4f627ebf52b061681a7a8b106daef3e79a1d/ragna/assistants/_api.py#L32-L38

This PR removes it and all subclass simply implement `def answer` directly.